client/me/purchases/manage-purchase/payment-info-block.tsx

```diff
@@ -31,6 +31,16 @@ export default function PaymentInfoBlock( {
 			card.meta?.find( ( meta ) => meta.meta_key === 'is_backup' )?.meta_value
 	);
 
+	if ( ! isRechargeable( purchase ) && hasPaymentMethod( purchase ) ) {
+		return (
+			<PaymentInfoBlockWrapper>
+				<div className={ 'manage-purchase__no-payment-method' }>
+					<Icon icon={ warning } />
+					{ translate( 'You don’t have a supporting payment method to renew this subscription' ) }
```

Comment:
Could we re-use the text in the [similar case below](https://github.com/Automattic/wp-calypso/blob/c8b35df0d209e20a9b6365468cbc894fad320624/client/me/purchases/manage-purchase/payment-info-block.tsx#L105)? It's slightly different:

`You don’t have a payment method to renew this subscription`

The benefit to using an existing translation is that we don't have to submit new text to be localized by our translators, saving time and effort.

Reply:
That makes sense, now I know 🙌🏻